### DISCUSSION TITLE
[Translation] - Monika After Story translation into Spanish

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#7827] [general - translations]
Monika After Story translation into Spanish
Version: 0.11.5 and 0.11.6
<b>Translators: SadlyThresh, Papu, Martin H, DarkMrMewtho</b>

<b>Link of the unofficial page in Spanish:</b>
https://papuweb.wixsite.com/monikaafterstoryesp

Repository in Github:
https://github.com/SadlyThresh/MAS_ESP

<b>_Note: This is an unofficial, non-profit translation, not affiliated with the Monika After Story development team or related to Team Salvato._</b>

<b>En Español:</b>
Traducción de Monika After Story al español
Versión: 0.11.5 y 0.11.6
<b>Traductores: SadlyThresh, Papu, Martin H, DarkMrMewtho</b>

<b>Link del sitio no oficial en Español:</b>
https://papuweb.wixsite.com/monikaafterstoryesp

Repositorio en Github:
https://github.com/SadlyThresh/MAS_ESP

<b>_Nota: Esta es una traducción no oficial, sin ánimo de lucro, no afiliada al equipo de desarrollo de Monika After Story o relacionada con el Equipo Salvato._</b>

**Comment by 20RaVen23** [comment]
👏 👏 👏

**Comment by ghost** [comment]
Monika After Story v0.11.8 **[ESPAÑOL]**

_The Spanish version has been updated.
You can download the update from our unofficial page_

**Link from the unofficial site:**

https://papuweb.wixsite.com/monikaafterstoryesp

**En español:**

_La versión en Español ha sido actualizada.
Puedes descargar la actualización desde nuestra página no oficial_

**Link del sitio no oficial:**

https://papuweb.wixsite.com/monikaafterstoryesp

**Comment by 20RaVen23** [comment]
I'm just going to link the main submod, spritepack, translations, etc. issue here.

#3765

**Comment by ghost** [comment]
Monika After Story v0.12.2 **[ESPAÑOL]**

_The Spanish version has been updated.
You can download the update from our unofficial page_

**Link from the unofficial site:**

https://papuweb.wixsite.com/monikaafterstoryesp

**En español:**

_La versión en Español ha sido actualizada.
Puedes descargar la actualización desde nuestra página no oficial_

**Link del sitio no oficial:**

https://papuweb.wixsite.com/monikaafterstoryesp

**Comment by Whispererboy141** [comment]
Does anybody know if the mod in DDSC is the actual más version for Android and PC?

**Reply by Just3090** [reply]
Sigues aquí?
